alts/src/main/java/io/grpc/alts/internal/AltsHandshakerStub.java

```diff
@@ -72,6 +74,14 @@ public HandshakerResp send(HandshakerReq req) throws InterruptedException, IOExc
     return result.get();
   }
 
+  /** Create a new writer if the writer is null. */
+  private synchronized void createWriterIfNull() {
```

Comment:
Why is this synchronized? I don't think this class can be used from multiple callers/threads.

Reply:
You are right. I was just being cautious :)  synchronized removed.